### PR TITLE
Adds in a bunch of throw text alerts

### PR DESCRIPTION
**Pull request by ProfessorPopoff**
## About The Pull Request

I've added a decent chunk of things for the new text alert system. They look pretty good!

image
image
image

I've applied it to things like addiction, overdoses, certain drugs, crafting, buying, selling, and a few other things. Go look!

I wasn't super thorough yet just because i'm blanking a bit on what else should have it. It can be expanded on more in the future. 

Looks very nice though! Big shoutout to our brother Killa, for #2353

### CODE REVIEW

File: code/datums/components/crafting/recipes.dm [hunk 1]
```diff
@@ -44,6 +44,7 @@
 	return TRUE
 
 /datum/crafting_recipe/proc/on_craft_completion(mob/user, atom/result)
+	user.throw_alert_text(/atom/movable/screen/alert/text/smallhappy, "[result.name] has been crafted successfully.", override = FALSE) // MOJAVE SUN EDIT - FO text alert
```
Comment (TheRealScarHomie) [inline code/datums/components/crafting/recipes.dm:47]: throw something in there so the name gets capitalized
image

Comment (ProfessorPopoff) [reply code/datums/components/crafting/recipes.dm:47]: yeah like a capital 9mm pistol...

Comment (TheRealScarHomie) [reply code/datums/components/crafting/recipes.dm:47]: just slap a "The" at the start of it

Comment (ProfessorPopoff) [reply code/datums/components/crafting/recipes.dm:47]: **The melt down gold bar has been crafted successfully.**

**Comment by Hekzder** [comment]
yea

**Pull request merged**